File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1036.py
from math import sqrt

n = [float(i) for i in input().split()]
a = n[0]
b = n[1]
c = n[2]

delta = b**2-4*a*c

if delta<0 or a==0:
    print("Impossivel calcular")
else:
    r1 = (-b+sqrt(delta))/(2*a)
    r2 = (-b-sqrt(delta))/(2*a)
    print("R1 = {:.5f}\nR2 = {:.5f}".format(r1, r2))

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1036.cs
using System;

class URI
{
    static void Main() {
        string an = System.Console.ReadLine().Trim();
        string[] n = an.Split(' ');
        double a = double.Parse(n[0]), b = double.Parse(n[1]), c = double.Parse(n[2]);
        double delta = b*b-4*a*c;
        if(delta<0 || a==0){ 
            Console.Write("Impossivel calcular\n");
        }
        else{ 
            Console.Write("R1 = {0:f5}\nR2 = {1:f5}\n", (-b+Math.Sqrt(delta))/(2*a), (-b-Math.Sqrt(delta))/(2*a));
        }
    }
}